### PR TITLE
screens and huge readme

### PR DESCRIPTION
**Pull request by jeffscottward**

### CODE REVIEW

File: src/utilities/uportSetup.js [hunk 1]
```diff
@@ -1,12 +1,9 @@
-import { Connect, SimpleSigner, QRUtil } from 'uport-connect'
+import { Connect, SimpleSigner } from 'uport-connect'
 
-const uport = new Connect('CryptoX', {
+const uport = new Connect('uPort Demo', {
```
Comment (pelle) [inline src/utilities/uportSetup.js]: See you changed it here. Please disregard my comment in previous PR.

File: src/components/CollectCredentials.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ class CollectCredentials extends Component {
     uport.attestCredentials({
       sub: this.props.uport.address,
       claim: {
-        name: this.props.uport.name,
+        Name: this.props.uport.name,
```
Comment (pelle) [inline src/components/CollectCredentials.js]: @jeffscottward claims should be lower case and each attestation should only contain one claim

Comment (pelle) [inline src/components/CollectCredentials.js]: You would really need to attest all 3 separately. You could either do it sequentially or wrap them all in `Promise.all`

Comment (jeffscottward) [inline src/components/CollectCredentials.js]: Ok then I will drop it for now. will make the lowercase change

**Pull request merged**